### PR TITLE
Integrate exchange page

### PR DESCRIPTION
**Pull request by meriadec**
2018-05-13_1944x1160

There is a problem with the coinhouse logo exported from zeplin, guess it's some svg stuff:

2018-05-13_2634x695

### CODE REVIEW

File: src/components/ExchangePage/logos/coinhouse.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import React from 'react'
+
+export default ({ width }: { width: number }) => (
```
Comment (gre) [inline src/components/ExchangePage/logos/coinhouse.js]: make it pure or external const (can wrap a `<g>`) ;)

File: src/icons/Exchange.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import React from 'react'
+
+export default ({ size, ...p }: { size: number }) => (
+  <svg viewBox="0 0 16 16" height={size} width={size} {...p}>
+    <path
```
Comment (gre) [inline src/icons/Exchange.js]: same, can be a const

Comment (meriadec) [reply src/icons/Exchange.js]: Maybe we should consider doing a pass on all icons to do that.

File: src/components/ExchangePage/logos/coinhouse.js (hunk 1 above)
Comment (meriadec) [reply src/components/ExchangePage/logos/coinhouse.js]: or use fragment eheh

**Pull request merged**